src/lsp.ts

```diff
@@ -43,19 +44,19 @@ export function activateLsp(
       };
       const clientOptions: LanguageClientOptions = {
         documentSelector: [{ scheme: "file" }],
-        outputChannel,
+        outputChannel: logger.getOutputChannel(),
       };
       client = new LanguageClient(
         "dprint",
         serverOptions,
         clientOptions,
       );
       await client.start();
+      resourceStores.push(client);
```

Comment:
```suggestion
      resourceStores.push(client);
      await client.start();
```

I think probably this to ensure that dispose still happens on failure?